kyverno-pss-policy-exceptions/README.md

```diff
@@ -0,0 +1,167 @@
+
+### Giant Swarm Implementation
+
+Each Giant Swarm project currently contains a PSP.
+If the application requires permissions not allowed by our security policies, the application's PSP will be directly replaced by one or multiple `PolicyException` resources.
+If the application does not require any elevated permissions, it no longer requires any policy resources to be contained in the repository. The PSP will be removed and no replacement will be added.
+
+The first generation of Kyverno PSS-controlled Giant Swarm management clusters will be managed as described in the Developer-Set Exceptions example below. In short: each project deploys its own exceptions in management clusters and workload clusters.
+
+Future improvements to the security platform intend to centralize the exceptions and eliminate the need to maintain individual application-level resources, moving us to a Centralized Exception model.
+
+### Workload Cluster Default Implementation
+
+We currently include a default restricted PSP in each workload cluster. This will be replaced by Kyverno PSS policies, enforced in `restricted` mode.
+
+Customers will have great flexibility in how they deliver `PolicyExceptions` to the cluster for any workloads which are not compliant with the `restricted` standard. Examples of possible exception models are included below.
+
+A newly-created workload cluster will contain `PolicyException` resources deployed for our applications, described above.
+
+### Migration Implications
+
+If this approach is accepted, we expect the migration to involve the following effort from Giant Swarm and our customers.
+
+Giant Swarm:
+
+- creation of `PolicyExceptions` for our applications which require them and can't be made compliant
+  - Shield has already made many workloads compliant so they don't require exceptions. Others can still be made compliant.
+  - this work could be done in parallel with removal / gating of PSP resources which will need to be removed in post-v1.25 clusters anyway
+- (Shield) creation of assistive guides. This is already in progress in parallel, but requires agreement on this RFC to be finalized
+
+Customers:
+
+- creation of `PolicyExceptions` for customer workloads
+  - this effort depends significantly on the customer's current model of PSP usage and application behavior
+    - for example, we have customers who centrally control all PSPs and customers who allow teams to control their own. Effort for the second is likely to be higher, also from a future management perspective
+
+Shield's experience with early Kyverno adopters is that organizations have drastically different ability and willingness to enforce security policies.
+We will provide resources based on what we already know from early adopters, what we learn from Giant Swarm's migration, and the unknown needs that we will inevitably learn from customers as they begin the move.
+
+### Migration Path
```

Comment:
cc @T-Kukawka as the timeline will need some v19 minor release
cc @Rotfuks as the timeline might need coordination with 1.25 in lifecycle team

Reply:
See also https://github.com/giantswarm/giantswarm/issues/26694